openedx/core/djangoapps/programs/tasks/v1/tasks.py

```diff
@@ -58,35 +59,35 @@ def get_completed_programs(student):
         student (User): Representing the student whose completed programs to check for.
 
     Returns:
-        list of program ids
+        list of program UUIDs
 
     """
-    meter = ProgramProgressMeter(student)
 
-    return meter.completed_programs
+    programs = [program['uuid'] for program in get_programs(student)]
```

Comment:
Now I am continuing to use the ProgramProgressMeter and passing a flag. 
I believe you suggested using a conditional on Friday?